### DISCUSSION TITLE
Is there a replacement for the mogrify command?

### DISCUSSION THREAD

**Discussion by zBernie** [ImageMagick/ImageMagick#5072] [Help]
I'm using ImageMagick 7.1.0-27 under Windows.  I was researching how to batch convert numerous .jpg files to .pdf, and found that I had to re-install IM and check the box for legacy items, which I did.  I was then able to run the command below to perform the aforementioned batch conversion using this command:

mogrify -format pdf *.jpg

My question is, is there a way to perform batch conversions without the legacy command mogrify?

**Comment by fmw42** [comment]
Under IM 7, normal install, not legacy IM 6, you would use

`magick mogrify -format pdf *.jpg`

**Reply by zBernie** [reply]
Great, thank you!

**Reply by fmw42** [reply]
All tools in ImageMagick 7 are prefaced by "magick"  (from their ImageMagick 6 versions), except for "convert" which is replaced by "magick"

**Reply by zBernie** [reply]
Thank you for that information.
